packages/dd-trace/test/appsec/iast/analyzers/resources/hardcoded-secrets-suite.json

```diff
@@ -3,267 +3,267 @@
     },
     {
       "id": "shopify-shared-secret",
-      "samples": ["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"]
+      "samples": [["shpss_AC5f", "50c52C4DCDf69aC52893AF369A74"]]
     },
     {
       "id": "slack-app-token",
-      "samples": ["xapp-2-TQRU-2-pi9b"]
+      "samples": [["xapp-4-J22", "2-45492-lh9a6"]]
     },
     {
       "id": "slack-bot-token",
-      "samples": ["xoxb-98990225585-12353152514"]
+      "samples": [["xoxb-25169", "65025178-629147915426"]]
     },
     {
       "id": "slack-config-access-token",
-      "samples": ["xoxe.xoxb-9-PMWS1VD0PB4OA3DK5YFEHVCZTBHO5S70C0W8QCFXX2AQ0WH8L8DMO223K68ZNY56EDMKHF42SSQPUV974JD6PULK73YTDQJ86EMWJ71FEL36LNJY7B5EV9RT2CMNQKYCIRSPINX7R72D0O8UT9WCUEN7X23HUNZ3W1Y", "xoxe.xoxp-8-PTN8WYGN3ND14QYWSDEV2QTQMEI8R90FZ0UCZWRY0J9QE2TIEAIUYJA9A7VSQI64X7CD6W75WQ4SQDLTV93R0GRPZUB1B9HSNF3G0PLSDN8HPST7VB6QOODRQWNMUF4WILELJE3GYLONHJAHBEG9D0E8BM0EB887SM8DNH"]
+      "samples": [["xoxe.xoxb-", "6-LF0T5F67P5IWYCH24XT8IB7TKICBNV808OGTTI7XDRCWYXIJ9HGJQYJKHHJWC4U47RYK6YJVZ6518L4XA3CFPZKJE5LL9KUJ7IYG98MK8UTU3SM6MVWI1OPIHOFAM7WOLW30WYXCQ1H4FZPWTP8NYLRS2Q8U3L8A8FIHZ"], ["xoxe.xoxp-", "2-7U2AGVCS8TR79I0AEK8G0DIE3H6ARI67GKPDTRLHNQVJ4BB0OIDOXD74LXG9ZCA0WT0Y2WQX812KR9K2GGVJFSKDMV26KMSLP5XLKM722PP6KXMMNO3TQ1VDJXLH4TMWJQ4WL8R3I3RZCN2ZIPT5S85XJTCXA08IC2X42"]]
     },
     {
       "id": "slack-config-refresh-token",
-      "samples": ["xoxe-4-02DA0JGWGLP95TSHFK7OYEZMQ45K733RZ0M8B7KCTS5XS8X3Q5I70CTWC0U6N6YNR8JFGTGH944GIC4RHC3BU61XG05LNNZX3RGUYRW7YA75HNNA5CZLCX2UHW9PX54WFHN16K2ZP6MU95Q3N5"]
+      "samples": [["xoxe-0-79K", "XK6A54W8OARDKVGM1643S5IWK086SOE8TYINNSH3O1TIVZNIVNJRDF21BJ0E5ZQ95SUVN39IZAP0LAAVNZKEFX5N6QLD1C7YLUMCE152OA2C0BQBJF1GPL4IXGZ6R5QXD1P0Y7QV3RZE53W"]]
     },
     {
       "id": "slack-legacy-bot-token",
-      "samples": ["xoxb-2635165305-qYktSa9LhLqa2jsoJuk"]
+      "samples": [["xoxb-93863", "72971490-tHHi79NSwtRjv5LSxZH15la1uv"]]
     },
     {
       "id": "slack-legacy-token",
-      "samples": ["xoxs-6-8-0-E3bc3"]
+      "samples": [["xoxo-8-038", "06-92105-4"]]
     },
     {
       "id": "slack-legacy-workspace-token",
-      "samples": ["xoxa-0-syVLBd4iVIadCxpG9rPMLP1FvrRs11UmaulCnXp", "xoxr-wPFyQFwKOTbIkg0DwggalChTNO28bNjc23fW9tO"]
+      "samples": [["xoxa-4-yvu", "xRbbzJ"], ["xoxr-EchWx", "U9oknZJBpb28wGNCtcGNNdCuKq8EYTFEesWbW"]]
     },
     {
       "id": "slack-user-token",
-      "samples": ["xoxe-6679527551-6986334739-864697748900-426-jwW3zxCLz3VcIsJq-ZgF8t8UdsfK"]
+      "samples": [["xoxp-30723", "4221103-4883165294-800633518510-HjrcUaoMJ4dMCu7OJACJkzqwL2WL"]]
     },
     {
       "id": "slack-webhook-url",
-      "samples": ["https://hooks.slack.com/services/a5ixwCnZkBl/sRAEeDcbQP+JDmyJpdtNgypWZYuyNgi", "https://hooks.slack.com/workflows/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"]
+      "samples": [["https://ho", "oks.slack.com/services/i6l82GnghZ6EvnrTIp+5HRrsyQuvtpDFuQpCmHMuSk0aI"], ["https://ho", "oks.slack.com/workflows/JTk448L1kQO4JlX4z4W0Vf+psAcPnidRkNHbL1xG54rYz"]]

```

Comment:
I guess that the most important thing to cut here is the token, which is maintained. And, why are the URLs different?

Reply:
but the regex looks for the url